cli.js

```diff
@@ -44,6 +44,9 @@ if (argv.ts) {
     docs = docs.use(/\.tsx?$/, new TypescriptPlugin({
         excludePaths: ["__tests__"],
     }));
+    docs = docs.use(/\.tsx?$/, new TypedocPlugin({
+        excludePaths: ["__tests__"],
+    }));
```

Comment:
why run two plugins on `.tsx?` files? seems like we should remove the original `TypescriptPlugin`